components/ActionSheet/src/MDCActionSheetController.h

```diff
@@ -303,4 +303,16 @@ typedef void (^MDCActionSheetHandler)(MDCActionSheetAction *_Nonnull action);
  */
 @property(nonatomic, copy, nullable) UIColor *tintColor;
 
+/**
+ The color of the divider at the top of the action.
+ */
+@property(nonatomic, copy, nullable) UIColor *dividerColor;
```

Comment:
Tip: Make this `nonnull` since it can be disabled entirely using `showsDivider` and even set to `clearColor`.

Reply:
Done